attributes/master.rb

```diff
@@ -135,6 +135,11 @@
   master['use_system_accounts'] = true
 
   #
+  # The groups of user under which Jenkins is running. Works for runit only.
+  #
+  master['groups'] = [node['jenkins']['master']['group']]
```

Comment:
Why do we need to introduce a completely new attribute here? Couldn't we just use the already defined `node['jenkins']['master']['group']` attribute?